packages/clerk-js/src/core/resources/Organization.ts

```diff
@@ -169,6 +170,7 @@ export class Organization extends BaseResource implements OrganizationResource {
     this.publicMetadata = data.public_metadata;
     this.membersCount = data.members_count;
     this.pendingInvitationsCount = data.pending_invitations_count;
+    this.adminDeleteEnabled = data.admin_delete_enabled;
```

Comment:
I kind of snuck this addition in here - the original PR was just too add these new properties as they were finished in the backend, but it expanded to covering the user delete feature. We decided to move the other two paired features, allowing admins to delete organizations, and gating organization creation permissions for users, to separate PRs, which I will be submitting soon.

If you want, I can move these out to the other PRs to keep the history squeaky clean, just let me know.

Reply:
We can leave it as is. No need for extra work.